src/server/handlers.rs

```diff
@@ -44,6 +72,7 @@ impl ReadHandler {
             status: status,
             datastore: datastore,
             sensor_specs: sensor_specs,
+            status_handlers: vec![],
```

Comment:
OK, so right now we provide the modifier, but don't enable it by default, right?